scripts/test-helpers

```diff
@@ -442,6 +443,7 @@ provision-agent() {
         --privileged \
         -e RKE2_TOKEN=$(cat $TEST_DIR/metadata/secret) \
         -e RKE2_URL=$RKE2_URL \
+        -e RKE2_DEBUG=true \
```

Comment:
Is this a testing remnant?

Reply:
No, I'd like to get this in. For some reason we start up the CI `rke2 server` instance with RKE2_DEBUG but not the `rke2 agent`, which means the logs from agent failures are missing a lot of useful info.